packages/package_info_plus/package_info_plus/pubspec.yaml

```diff
@@ -29,18 +29,18 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  http: ^0.13.0
-  meta: ^1.3.0
-  path: ^1.8.0
+  http: ^0.13.5
+  meta: ^1.8.0
+  path: ^1.8.2
   package_info_plus_platform_interface: ^2.0.1
-  win32: ">=2.7.0 <5.0.0"
+  win32: ^4.1.3
```

Comment:
As the author of `win32`, I'd recommend against setting such a wide set of constraints as was originally the case, but equally the replacement is unnecessarily narrow. Perhaps `^4.0.0` would be better?

Reply:
All the breaking changes in the 4.x series are to the experimental WinRT projection. The other changes are just adding new APIs, so ^4.0.0 should be great!

Reply:
Alright, thanks for explanations.